sdk/nodejs/src/__mocks__/handlers.ts

```diff
@@ -4,7 +4,11 @@ import { rest } from 'msw'
 export const handlers = [
     rest.post('https://bucketing-api.devcycle.com/v1/variables/test-key-not-in-config', (req, res, ctx) => {
         return res(
-            ctx.status(400)
+            ctx.status(404),
+            ctx.json({
+                statusCode: 404,
+                message: 'Feature not found for key: test-key-not-in-config'
```

Comment:
variable?

Reply:
oh yeah i fixed this in the actual api, it doesn't get used anymore here so i'll just take it out the the mock